### PR TITLE
CLOUD-674 add info about build platform

### PR DESCRIPTION
**Pull request by nmarukovich**
[CLOUD-674](https://jira.percona.com/browse/CLOUD-674) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| rs-shard-migration | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| pitr | failed |
| users | passed |
| data-sharded | passed |
| non-voting | failed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/7a41b31363944435517b92802c082831a6775918
image: `perconalab/percona-server-mongodb-operator:PR-864-7a41b313`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | failed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| demand-backup | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| smart-update | passed |
| version-service | passed |
| demand-backup-sharded | failed |
| users | failed |
| upgrade | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/b97c7d5819ee8c1c91403af0108d5f076729bfa3
image: `perconalab/percona-server-mongodb-operator:PR-864-b97c7d58`

**Comment by tplavcic** [comment]
Btw. if we will add this to PSMDB probably we should add it for all.

**Comment by nmarukovich** [comment]
> Btw. if we will add this to PSMDB probably we should add it for all.

yep! I will add to pxc with cross site tests) In other operators I can add after approving of this one)

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| rs-shard-migration | passed |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| storage | passed |
| users | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | failed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| data-sharded | passed |
| non-voting | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | failed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/b97c7d5819ee8c1c91403af0108d5f076729bfa3
image: `perconalab/percona-server-mongodb-operator:PR-864-b97c7d58`

**Review by tplavcic — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| init-deploy | passed |
| storage | passed |
| version-service | passed |
| limits | passed |
| self-healing | passed |
| users | passed |
| scaling | passed |
| self-healing-chaos | passed |
| data-sharded | passed |
| security-context | passed |
| operator-self-healing | passed |
| non-voting | passed |
| rs-shard-migration | passed |
| operator-self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | failed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/b97c7d5819ee8c1c91403af0108d5f076729bfa3
image: `perconalab/percona-server-mongodb-operator:PR-864-b97c7d58`

**Review by hors — CHANGES_REQUESTED** [review]
I think we need to have possibly to change it via env variable.

**Comment by nmarukovich** [comment]
> I think we need to have possibly to change it via env variable.

Any problem! In this case I can add info in the test instructions about the variable
""export DOCKER_DEFAULT_PLATFORM=linux/amd64

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/90693969687dc295681e1dc5e61be7c408edfc7b
image: `perconalab/percona-server-mongodb-operator:PR-864-90693969`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/90693969687dc295681e1dc5e61be7c408edfc7b
image: `perconalab/percona-server-mongodb-operator:PR-864-90693969`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| arbiter | passed |
| self-healing-chaos | passed |
| service-per-pod | passed |
| operator-self-healing | passed |
| liveness | passed |
| operator-self-healing-chaos | passed |
| smart-update | passed |
| version-service | passed |
| init-deploy | passed |
| users | passed |
| limits | passed |
| data-sharded | passed |
| scaling | passed |
| non-voting | passed |
| security-context | passed |
| rs-shard-migration | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/b97c7d5819ee8c1c91403af0108d5f076729bfa3
image: `perconalab/percona-server-mongodb-operator:PR-864-b97c7d58`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | failed |
| limits | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| scheduled-backup | passed |
| version-service | passed |
| users | passed |
| demand-backup-sharded | failed |
| scaling | passed |
| security-context | passed |
| data-sharded | passed |
| rs-shard-migration | passed |
| non-voting | passed |
| upgrade | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/8ca07e4b76e70772acf307d12411474d9d6a07d2
image: `perconalab/percona-server-mongodb-operator:PR-864-8ca07e4b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | failed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| security-context | passed |
| arbiter | passed |
| rs-shard-migration | passed |
| scheduled-backup | passed |
| service-per-pod | failed |
| demand-backup-sharded | failed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| data-sharded | passed |
| upgrade-sharded | passed |
| non-voting | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/c3d3447a78909beded30eaefb04edcee78729f7f
image: `perconalab/percona-server-mongodb-operator:PR-864-c3d3447a`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | failed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/90693969687dc295681e1dc5e61be7c408edfc7b
image: `perconalab/percona-server-mongodb-operator:PR-864-90693969`

File: e2e-tests/build [hunk 1]
```diff
@@ -21,6 +21,7 @@ build_operator() {
 	fi
 
 	export IMAGE
+	export DOCKER_DEFAULT_PLATFORM=linux/amd64
```
Comment (delgod) [inline e2e-tests/build:24]:
```suggestion
	export DOCKER_DEFAULT_PLATFORM=${DOCKER_DEFAULT_PLATFORM:-linux/amd64}
```

**Review by delgod — APPROVED** [review]

**Review by hors — DISMISSED** [review]
@nmarukovich please update README.md too. Thanks

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| monitoring-2-0 | passed |
| limits | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| liveness | passed |
| demand-backup-sharded | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| data-sharded | passed |
| upgrade-sharded | passed |
| non-voting | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/38b01df61228b60a45f02672619f972b2486e06d
image: `perconalab/percona-server-mongodb-operator:PR-864-38b01df6`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| scheduled-backup | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| demand-backup-sharded | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| users | passed |
| upgrade-sharded | passed |
| pitr | passed |
| data-sharded | passed |
| non-voting | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/b609010579d2643c8979cfb59c4d8713cab84e57
image: `perconalab/percona-server-mongodb-operator:PR-864-b6090105`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| one-pod | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| demand-backup-sharded | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| data-sharded | passed |
| pitr | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/bc5ac5ab664e32f0062fbb17570ea5933ca8792d
image: `perconalab/percona-server-mongodb-operator:PR-864-bc5ac5ab`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| scheduled-backup | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| liveness | passed |
| demand-backup-sharded | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/864/commits/00f45904a94ef790dea1ee1b00489d0be97a8ec4
image: `perconalab/percona-server-mongodb-operator:PR-864-00f45904`

**Review by delgod — APPROVED** [review]

**Pull request merged**